lib/modules/platform/gitlab/index.ts

```diff
@@ -694,7 +694,7 @@ export async function mergePr({ id }: MergePRConfig): Promise<boolean> {
 export function massageMarkdown(input: string): string {
```

Comment:
This change also breaks the normal GitLab Merge Request vocabulary:

```diff
 modules/platform/gitlab/index › massageMarkdown(input) › returns updated pr body

expect(received).toMatchSnapshot()

Snapshot name: `modules/platform/gitlab/index massageMarkdown(input) returns updated pr body 1`

- Snapshot  - 1
+ Received  + 1

@@ -1,8 +1,8 @@
  https://github.com/foo/bar/issues/5 plus also [a link](https://github.com/foo/bar/issues/5

-   Merge Requests are the best, here are some MRs.
+   Merge Requests are the best, here are some PRs.

    ## Open

  These updates have all been created already. Click a checkbox below to force a retry/rebase of any.
  ↵

    at Object.<anonymous> (lib/modules/platform/gitlab/index.spec.ts:2099:46)
    at runMicrotasks (<anonymous>)
```

Reply:
Good catch, is `PRs` the only example you can think of which we'd want to replace?

Reply:
We may want to search/replace these, when Renovate detects the GitLab platform?

- `Pull Request` -> `Merge Request`
- `Pull Requests` -> `Merge Requests`
- And the lowercase variants

Reply:
Reminder: we ideally only want this term changed _in our own text_. i.e. words we've created. Changing terms in changelogs is an unfortunately side effect

Reply:
> We may want to search/replace these, when Renovate detects the GitLab platform?
> 
> * `Pull Request` -> `Merge Request`
> * `Pull Requests` -> `Merge Requests`
> * And the lowercase variants

Thanks for the feedback 😄, so for this case should I change the regex [on this line](https://github.com/renovatebot/renovate/blob/35.2.0/lib/modules/platform/gitlab/index.ts#L710)
From
`replace(regEx(/Pull Request/g), 'Merge Request')`
To
`replace(regEx(/[Pp][Uu][Ll][Ll] [Rr][Ee][Qq][Uu][Ee][Ss][Tt]/g), 'Merge Request')`?

And am I allowed to update the current test cases to match the proposed fix? Thanks a lot before

Reply:
I don't think we have to worry about "false positives" for the string `Pull Request` like we do for `PR`